mp/src/game/server/momentum/mom_fx.h

```diff
@@ -0,0 +1,25 @@
+//====== Copyright � 1996-2005, Valve Corporation, All rights reserved. =======
+//
+//
+//
+//=============================================================================
+#pragma once
+
+#include "particle_parse.h"
+#include "weapon/mom_weapon_parse.h"
```

Comment:
Are these needed here? Remove these, and forward declare where possible.

Reply:
These are needed because the methods reference `ParticleAttachment_t` and `CWeaponID` which are declared in those headers respectively.